tests/test_epos.py

```diff
@@ -0,0 +1,90 @@
+from impy.kinematics import CenterOfMass
+from impy.models import EposLHC
+from impy.constants import GeV
+import numpy as np
+from numpy.testing import assert_allclose
+from .util import (
+    reference_charge,
+    run_in_separate_process,
+    skip_on_ci_if_model_is_incompatible,
+)
+import pytest
+from particle import literals as lp
+from functools import lru_cache
+
+skip_on_ci_if_model_is_incompatible(EposLHC)
+
+
+def run_pp_collision():
+    evt_kin = CenterOfMass(10 * GeV, "proton", "proton")
+    m = EposLHC(evt_kin, seed=4)
+    m.set_stable(lp.pi_0.pdgid, True)
+    for event in m(1):
+        pass
+
+    # some methods only work on original event
+    assert event.impact_parameter > 0
+    assert event.n_wounded_A == 1
+    assert event.n_wounded_B == 1
+    assert event.n_wounded == 2
+
+    return event
+
+
+@pytest.fixture
+@lru_cache(maxsize=1)  # EposLHC initialization is very slow
```

Comment:
It shouldn't be very slow. At least not in my tests

Reply:
Hm, I think that's a copy and paste bug, can be removed, but it also doesn't hurt to have the lru_cache.